mysql-test/columnstore/basic/r/MCOL-5568-out-of-range.result

```diff
@@ -0,0 +1,16 @@
+DROP DATABASE IF EXISTS MCOL5568;
+CREATE DATABASE MCOL5568;
+USE MCOL5568;
+CREATE TABLE test_mult (
+indemnity_paid INT(11),
+n_clms TINYINT(3) UNSIGNED
+) ENGINE=COLUMNSTORE;
+INSERT INTO test_mult (indemnity_paid, n_clms) VALUES (-10, 1);
+SELECT indemnity_paid, n_clms, indemnity_paid * n_clms FROM test_mult;
+ERROR HY000: Internal error: MCS-2053: The result is out of range for function "*" using value(s): -10.000000 1.000000
+SELECT indemnity_paid, n_clms, indemnity_paid + n_clms FROM test_mult;
+ERROR HY000: Internal error: MCS-2053: The result is out of range for function "+" using value(s): -10.000000 1.000000
+SELECT indemnity_paid, n_clms, (indemnity_paid + 9) + n_clms FROM test_mult;
```

Comment:
I would appreciate if you extend the code and the examples for combinations of: smaller unsigned SQL datatypes, e.g. UBINT, USMALLINT, UTINYINT, *,/ operations and a signed type as the second argument.